examples/medplum-patient-intake-demo/data/core/valuesets.json

```diff
@@ -0,0 +1,182 @@
+{
+  "resourceType": "Bundle",
+  "type": "transaction",
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:97139871-af48-46af-a8b4-ed16fa8b6b6f",
+      "request": {
+        "method": "PUT",
+        "url": "ValueSet?name=pregnancy-status-codes"
+      },
+      "resource": {
+        "resourceType": "ValueSet",
+        "status": "active",
+        "url": "http://example.com/pregnancy-status",
+        "name": "pregnancy-status-codes",
+        "title": "Pregnancy Status Codes",
+        "description": "These codes are used to indicate pregnancy status.",
+        "compose": {
+          "include": [
+            {
+              "system": "http://snomed.info/sct",
+              "concept": [
+                {
+                  "code": "146799005",
+                  "display": "Possible pregnancy"
+                },
+                {
+                  "code": "60001007",
+                  "display": "Not pregnant (finding)"
+                },
+                {
+                  "code": "77386006",
+                  "display": "Pregnancy"
+                }
+              ]
+            },
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v3-NullFlavor",
+              "concept": [
+                {
+                  "code": "UNK",
+                  "display": "Unknown"
+                }
+              ]
+            }
+          ]
+        }
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:6abaa6c8-d822-46a4-b3a2-70639475e5ae",
+      "request": {
+        "method": "POST",
+        "url": "ValueSet?name=us-core-usps-state",
+        "ifNoneExist": "url=http://hl7.org/fhir/us/core/ValueSet/us-core-usps-state"
+      },
+      "resource": {
+        "resourceType": "ValueSet",
+        "status": "active",
+        "version": "0.0.0",
+        "url": "http://hl7.org/fhir/us/core/ValueSet/us-core-usps-state",
+        "name": "us-core-usps-state",
+        "expansion": {
+          "timestamp": "2024-08-12T00:00:00.000Z",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/valueset-expansionSource",
+              "valueUri": "http://hl7.org/fhir/us/core/ValueSet/us-core-usps-state"
+            }
+          ],
```

Comment:
@rahul1 This addresses State ValueSet https://github.com/medplum/medplum/pull/5026#issuecomment-2283766417